packages/shared/src/types.ts

```diff
@@ -115,4 +115,7 @@ export interface NotificationMap {
     uri: string;
     types: TypegenData[];
   };
+  extensionError: {
```

Comment:
I'd probably rename this to be a little more specific - like maybe `extractionError` or something like that